udp-ballerina/build.gradle

```diff
@@ -219,8 +203,8 @@ task ballerinaBuild {
             }
         }
         copy {
-            from file("$project.projectDir/target/balo")
-            into file("$artifactCacheParent/balo/${packageOrg}/${packageName}/${tomlVersion}")
+            from file("$project.projectDir/target/bala")
+            into file("$artifactCacheParent/bala/${packageOrg}/${packageName}/${tomlVersion}")
```

Comment:
Why do we have to do the `balo` to `bala` change again?

Reply:
this is changed now